src/XPurchase/VirtoCommerce.XPurchase/Schemas/InputAddWishlistItemsType.cs

```diff
@@ -0,0 +1,13 @@
+﻿using GraphQL.Types;
+
+namespace VirtoCommerce.XPurchase.Schemas
+{
+    public class InputAddWishlistItemsType : InputObjectGraphType
+    {
+        public InputAddWishlistItemsType()
+        {
+            Field<NonNullGraphType<StringGraphType>>("listId");
+            Field<NonNullGraphType<ListGraphType<InputNewBulkItemType>>>("listItems", "Bulk list items");
```

Comment:
`InputNewBulkItemType` is for bulk order only and adds products by SKU, instead of product ID. please introduce new `InputAddWishlistItemsType` or rename command to `addWishlistBulkItems` if it's for bulk ordering and should work with SKU instead of product ID.